### DISCUSSION TITLE
小米路由器mini    CPU为MT7620A处理器  MIPS架构的   这种要下载哪一个ipk  文件呢？

### DISCUSSION THREAD

**Discussion by 86229199** [messense/aliyundrive-webdav#174] [Q&A] [answered]
小米路由器mini    CPU为MT7620A处理器  MIPS架构的   这种要下载哪一个ipk  文件呢？

**Comment by messense** [comment, marked as answer]
运行 opkg print-architecture 看看输出。

**Reply by 86229199** [reply]
谢谢，从下面这些信息可以看出吗？

**Reply by 86229199** [reply]
![1](https://user-images.githubusercontent.com/16441012/143969871-f721f3f0-ef11-4d2f-b5f0-f45cf9962b7b.jpg)

**Reply by messense** [reply]
https://github.com/messense/aliyundrive-webdav/releases/download/v1.0.0/aliyundrive-webdav_1.0.0_mipsel_24kc.ipk

**Reply by messense** [reply]
> 谢谢，从下面这些信息可以看出吗？

opkg print-architecture 才有准确的包名。

**Reply by 86229199** [reply]
大佬 您好，我按你说的命令执行了， 
执行命令：opkg print-architecture
得到：
arch all 1
arch noarch 1
arch ralink 10
这个要下载那个ipk的包呢？
另外我下载 aliyundrive-webdav_1.0.0_mipsel_24kc.ipk  这个包，安装会出现这个错误，不晓得是什么问题：
如下图：
![2](https://user-images.githubusercontent.com/16441012/144050882-f4958d5a-804c-403c-967d-b71e150df21a.png)

**Reply by messense** [reply]
`/etc/opkg.conf` 增加

```
arch all 1
arch noarch 1
arch ralink 10
arch mipsel_24kc 100
```

再安装试试。

**Reply by 86229199** [reply]
谢谢，按你添加的可以把ipk安装成功了，但是又出现服务启动不起来的问题， 路由器阿里云盘配置和错误日志如下：
![1](https://user-images.githubusercontent.com/16441012/144254067-c7b65ec3-cf48-4003-a81b-ead91700d01e.png)
![2](https://user-images.githubusercontent.com/16441012/144254084-32feed83-ab38-4ff3-9096-15c3809c4034.png)
![3](https://user-images.githubusercontent.com/16441012/144254088-f33cfe5c-9f80-4d62-9cef-38fc44bd8716.png)
![4](https://user-images.githubusercontent.com/16441012/144254095-6c03cc2e-fd2b-4ed4-a621-edce90cc9d47.png)

**Reply by messense** [reply]
系统太老了，证书有问题。

建议换个路由器。。。

**Reply by 86229199** [reply]
我用根据window  exe的包 ，用阿里云的那个  refesh_token  是可以正常启动的，  但是配置到路由器就启动不起来。

**Reply by 86229199** [reply]
是路由器的问题 是吧  好的  谢谢了 麻烦您了`
